src/Time/Units.hs

```diff
@@ -74,7 +74,7 @@ import Time.Rational (type (%), type (**), type (//), type (:%), DivRat, KnownRa
 import qualified Control.Concurrent as Concurrent
 
 newtype Time (rat :: Rat) = Time RatioNat
```

Comment:
Maybe we can also add record field for `Time` constructor? Like

```haskell
newtype Time (rat :: Rat) = Time { unTime :: RatioNat }
```
What do you think? Sometimes can be convenient to write functions in point-free style.